src/main/java/org/opensearch/performanceanalyzer/rca/framework/metrics/ReaderMetrics.java

```diff
@@ -106,7 +154,14 @@
                     Statistics.MIN,
                     Statistics.MEAN,
                     Statistics.COUNT,
-                    Statistics.SUM));
+                    Statistics.SUM)),
+
+    /**
+     * A blanket exception code for {@link
+     * org.opensearch.performanceanalyzer.reader.ReaderMetricsProcessor} failures.
+     */
+    OTHER("Other", "count", Collections.singletonList(Statistics.COUNT));
```

Comment:
Can we name this OTHER_READER_METRICS_PROCESSOR_ERRORS or something like this? Or atleast OTHER_ERRORS? Not sure how much info we will get from ```OTHER``` when we see them in prod.

Reply:
`ReaderMetricsProcessor` is one of the places this is used, and this is sort of meant as a blanket exception code. Will keep it like this for now and we can re-visit.